mage_ai/frontend/components/charts/LineSeries.tsx

```diff
@@ -455,9 +455,9 @@ function LineSeriesContainer({
 
   return (
     <>
-      <div style={{ display: 'flex', height: parentHeight, width: '100%', marginBottom: UNIT }}>
+      <div style={{ display: 'flex', height: parentHeight,  marginBottom: UNIT, width: '100%' }}>
```

Comment:
does `marginBottom` do anything if it's just `8` without `px` or any units?

Reply:
@nathaniel-mage

Reply:
That's not really part of my PR, so idk for sure, but I do know that without px it shouldn't be working, unless UNIT is a '8px' string instead of a number.